flink-java/src/test/java/org/apache/flink/api/java/operator/JoinOperatorTest.java

```diff
@@ -1211,7 +1210,6 @@ public String toString() {
 		public String myString;
 		public Object nothing;
 		public List<String> countries;
-		public Writable interfaceTest;
```

Comment:
`Writable` represents in this test just an interface. Can you replace it by another interface to maintain the coverage of this test?